src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -2375,6 +2438,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return OperationKind.None
         End Function
 
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation)
+            Get
+                Return ImmutableArray.Create(Of IOperation)(Me.OriginalArgument)
```

Comment:
>Return ImmutableArray.Create(Of IOperation)(Me.OriginalArgument) [](start = 16, length = 64)

I see no corresponding change in any tests, this means a test gap is introduced. Please close the gap.